### PR TITLE
add currenSiteLanguage and currentSiteId in method parameter of node …

### PR DESCRIPTION
**Pull request by itkg-alavieille**
in relation with : https://github.com/open-orchestra/open-orchestra-model-bundle/pull/299

### CODE REVIEW

**Comment by nicolasThal** [comment]
:+1:

File: ApiBundle/Controller/NodeController.php [hunk 1]
```diff
@@ -33,13 +33,16 @@ class NodeController extends BaseController
      */
     public function showAction(Request $request, $nodeId)
     {
-        $language = $request->get('language');
+        $currentSiteManager = $this->get('open_orchestra.manager.current_site');
+        $language = (null !== $request->get('language')) ? $request->get('language') : $currentSiteManager->getCurrentSiteDefaultLanguage();
```
Comment (itkg-ngilain) [inline ApiBundle/Controller/NodeController.php]: you use the @nicolasThal syntax : $request->get('language', $currentSiteManager->getCurrentSiteDefaultLanguage())

**Comment by itkg-alavieille** [comment]
@itkg-ngilain

**Comment by itkg-ngilain** [comment]
:+1:

**Pull request merged**